regulations/static/regulations/js/source/views/main/main-view.js

```diff
@@ -27,8 +27,11 @@ var MainView = Backbone.View.extend({
 
     initialize: function() {
 
+
+        this.dataTable = null;
```

Comment:
This handles multiple instances as well - the plugin maintains a collection.  I'll pluralize the variable name to make that clearer.